praw/helpers.py

```diff
@@ -111,6 +111,104 @@ def valid_redditors(redditors, sub):
             if resp['ok']]
 
 
+def all_submissions(reddit_session,
+                    subreddit,
+                    highest_timestamp=None,
+                    lowest_timestamp=None,
+                    verbosity=1):
+    """Yield submissions between two timestamps.
+
+    If both ``highest_timestamp`` and ``lowest_timestamp`` are unspecified,
+    yields all submissions in the ``subreddit``.
+
+    Submissions are yielded from newest to oldest(like in the "new" queue).
```

Comment:
I think reverse order makes much more sense if you're trying to download a whole subreddit, so this helper acts as sort of the "new" queue. However, maybe if you want submissions from a specific day or week, you'd like to get them in order from oldest to newest. 

I guess I should add a flag here?